packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -384,13 +355,180 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
         </div>
     </oui-step-form>
 
+    <!-- private network starts here -->
+    <oui-step-form
+        data-header="{{ :: 'pci_projects_project_instances_add_privateNetwork_title' | translate }}"
+        data-editable="!$ctrl.isLoading"
+        name="instances_add_private_network_configuration"
+        data-on-focus="$ctrl.onModeFocus()"
+        data-on-change="$ctrl.onModeChange()"
+        data-submit-text="{{ :: 'pci_projects_project_instances_add_privateNetwork_next' | translate }}"
+        data-valid="!$ctrl.isGatewayLoading"
+        data-on-submit="$ctrl.onModeSubmit()"
+        data-navigation="!$ctrl.enableModeSubmit() && !$ctrl.isGatewayLoading"
+    >
+        <p>
+            <strong
+                data-translate="pci_projects_project_instances_add_privateNetwork_sub_title"
+            ></strong>
+        </p>
```

Comment:
We can use Bootstrap class:

```suggestion
        <p class="font-weight-bold" data-translate="pci_projects_project_instances_add_privateNetwork_sub_title"></p>
```